ballista/rust/scheduler/src/planner.rs

```diff
@@ -59,15 +58,14 @@ impl DistributedPlanner {
     /// Returns a vector of ExecutionPlans, where the root node is a [ShuffleWriterExec].
     /// Plans that depend on the input of other plans will have leaf nodes of type [UnresolvedShuffleExec].
     /// A [ShuffleWriterExec] is created whenever the partitioning changes.
-    pub async fn plan_query_stages<'a>(
+    pub fn plan_query_stages<'a>(
```

Comment:
Is there some reason remove the `async `🤔
I think there are some io work in `plan_query_stages ` like save status in db

Reply:
I think it was doing that at tone time but the implementation now is not doing IO so I changed it back to sync.